tasks/audit-cat2.yml

```diff
@@ -77,3 +77,88 @@
       - RHEL-07-040640
       - always
       - ssh
+
+- name: "MEDIUM | RHEL-07-020840 | PATCH | All local initialization files for interactive users must be owned by the home directory user or root."
+  rhel7_020840: check=True
+  failed_when: no
+  changed_when: no
+  ignore_errors: yes
+  register: rhel_07_020840_audit
+  tags:
+      - cat2
+      - medium
+      - patch
```

Comment:
Should we be changing the permissions on the system?  So if .bashrc in /root is owned by apache, should we correct or just show the error?